pkg/webservice/handlers_test.go

```diff
@@ -921,6 +925,19 @@ func getNodesUtilByType(t *testing.T, nodesUtilList []*dao.NodesUtilDAOInfo, res
 }
 
 func TestPartitions(t *testing.T) {
+	schedulerContext = &scheduler.ClusterContext{}
+
+	var req *http.Request
+	req, err := http.NewRequest("GET", "/ws/v1/partitions", strings.NewReader(""))
+	assert.NilError(t, err, "App Handler request failed")
+
+	resp := &MockResponseWriter{}
+	var partitionInfo []*dao.PartitionInfo
+	getPartitions(resp, req)
+	err = json.Unmarshal(resp.outputBytes, &partitionInfo)
+	assert.NilError(t, err, unmarshalError)
+	assert.Equal(t, len(partitionInfo), 0)
```

Comment:
ditto, please ensure `partitionInfo` is not nil

Reply:
done